src/main/java/codeu/controller/AdminServlet.java

```diff
@@ -0,0 +1,77 @@
+package codeu.controller;
+import codeu.model.data.User;
+import codeu.model.store.basic.UserStore;
+
+import java.io.PrintWriter;
+import java.io.IOException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import javax.servlet.ServletException;
+
+import java.util.List;
+import java.util.Arrays;
+import java.util.Scanner;
+
+public class AdminServlet extends HttpServlet {
+
+	//Store clas that gives access to Users. 
+	private UserStore userStore;
+
+	// set state for request on server
+	public void init() throws ServletException {
+		super.init();
+		setUserStore(UserStore.getInstance());
+	}
+	
+	// set UserStore used by this servlet
+	void setUserStore(UserStore userStore) {
+		this.userStore = userStore;
+	}
+
+	
+	@Override
+	//this is to check for the users 
+
+	public void doGet (HttpServletRequest request, HttpServletResponse response) throws IOException, ServletException{
+		request.getRequestDispatcher("/login").forward(request, response);
```

Comment:
I'm not sure this line is what you want to do. The parameter you pass into `request.getRequestDispatcher()` should probably be a path to a JSP file (and it should probably not be the login page). Take a look at how the other servlets do it: https://github.com/costaa17/CodeU-Spring-2018/blob/master/src/main/java/codeu/controller/LoginServlet.java#L58